ci-operator/step-registry/ipi/upgrade/aws/ipi-upgrade-aws-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: ipi-upgrade-aws
```

Comment:
Can we drop this unused workflow?

Reply:
Yeah thinking about it more actually we NEED to use the origin e2e because that's where the other half of the upgrade happens ... so people are going to love trying to integrate with this